doc/rst/language/spec/classes.rst

```diff
@@ -1721,7 +1721,10 @@ omitted when the field access is within a method. In this case the
 receiver is the method’s receiver. The receiver clause can also be
 omitted when the field access is within a class declaration. In this
 case the receiver is the instance being implicitly defined or
-referenced.
+referenced. When the receiver clause is omitted, the compiler will
+consider the possibility that the identifier refers to a field after
+considering if it refers to a method formal argument but before
+considering if it refers to a variable in another parent scope.
```

Comment:
Would it read more clearly if the three possibilities were listed in the same order as their consideration? (Even though this particular section of the spec is about the middle one.)

Also, in the PR description, "indentifier".